templates/DataCollector/meilisearch.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 50 50" fill="none">
+    <path d="M0 24.4968L7.60256 5.04523C8.67392 2.30411 11.3161 0.5 14.2591 0.5H18.8426L11.24 19.9516C10.1687 22.6927 7.52653 24.4968 4.58348 24.4968H0Z" fill="url(#paint0_linear_2735_3490)"/>
```

Comment:
Where those `#paint*` are defined?

Reply:
I've used the [meilisearch website logo ](https://www.meilisearch.com/_next/static/media/logo.cd874c57.svg) and removed some data of it without taking much care on it for now as I may use another one at the end, perhaps meilisearch team do not want their logo but rather prefer a more generic one for "search"